src/rollup/widgets/construct-widget-layout.js

```diff
@@ -333,4 +334,8 @@ export default function constructWidgetLayout (widget, config) {
   if (config.msg) {
     widgetMessage.innerHTML = config.msg;
   }
+
+  if (config.footerText) {
+    widgetFooter.innerHTML = config.footerText;
```

Comment:
should be able to add lots of tests for this. one to ensure the div gets populated properly when there is text to drop in and another to ensure that it does not get added at all for widgets with no `footerText`. both on every type of widget it can effect. could also test empty string vs missing undefined key if you want to go wild.